db/dump.py

```diff
@@ -73,7 +72,8 @@ def dump_db(location, threads=None, incremental=False, dump_id=None):
     Returns:
         Path to created dump.
     """
-    create_path(location)
+    if not os.path.exists(location):
```

Comment:
Hm, interesting. If it doesn't actually improve anything, let's keep it as it was before. I just didn't know that `os.makedirs